components/sup/src/manager/mod.rs

```diff
@@ -291,18 +309,15 @@ impl Manager {
         Ok(())
     }
 
-    pub fn remove_service(&self, service: &Service) -> Result<()> {
-        // TODO FN: there is more to removing a service, more to follow
-        service.remove()?;
+    pub fn remove_service(&self, service: &mut Service) -> Result<()> {
```

Comment:
If we're going with load/unload we should think about updating the internal functions and references to load/unload as well. It forces the distinction between loading and starting (vs. unloading and stopping).